akka-typed/src/main/scala/akka/typed/ActorRef.scala

```diff
@@ -9,6 +9,13 @@ import language.implicitConversions
 import scala.concurrent.Future
 
 /**
+ * Helper parent to allow [[ActorRef]] to be used as key in TypedMultiMap.
```

Comment:
Aeh, nope, this is just left over from some work that has not survived—but I should actually include a test that this comment does indeed hold water.